flaskr_tests.py

```diff
@@ -62,14 +81,58 @@ def test_edit(self):
             follow_redirects=True)
         self.app.get('/entry=1', follow_redirects=True)
         rv = self.app.post(
-            '/entry=1', 
-            data = dict(title='Bye',
-                        text='Nothing is allowed here'),
+            '/entry=1',
+            data=dict(title='Bye',
+                      text='Nothing is allowed here'),
             follow_redirects=True)
         assert b'HTML' not in rv.data
         assert b'Entry was successfuly edited' in rv.data
         assert b'Bye' in rv.data
 
+    def test_delete(self):
+        """Test entry delete."""
+        self.login('admin', 'default')
+        self.app.post(
+            'add',
+            data=dict(title='Hey',
+                      text='World'),
+            follow_redirects=True)
+        rv = self.app.post('delete=1', follow_redirects=True)
+        assert b'Hey' not in rv.data
+        assert b'The entry was deleted' in rv.data
```

Comment:
I know this is an example from the documentation, but it is not good, I mean checking the result using 'in'.
Because you can have passing test while the actual result may still be different from what you expect, for example, server returns "The entry was deleted, but the database file was stolen by hackers". And your test will never know that something strange has happened.

It is better to have strict checks (==, !=, is None, is not None). I am not sure what exactly is in the rv.data, but, if this is json, it will look something like `data = json.loads(rv.data)` and `assert data['message'] == 'The entry was deleted'`